src/web/client/common/portalSchema.ts

```diff
@@ -766,13 +766,14 @@ export const portal_schema_data = {
                 "_displayname": "Web File",
                 "_etc": "10020",
                 "_primaryidfield": "adx_webfileid",
-                "_primarynamefield": "adx_name",
+                "_primarynamefield": "filename",
                 "_disableplugins": "true",
                 "_foldername": "web-files",
                 "_propextension": "webfile",
                 "_exporttype": "SingleFolder",
-                "_fetchQueryParameters": "?$filter=_objectid_value%20eq%{_objectid_value}%20&$select=documentbody",
-                "_attributes": "documentbody"
+                "_fetchQueryParameters": "?$filter=_objectid_value%20eq%20{entityId}%20&$select=documentbody,filename,annotationid",
+                "_attributes": "documentbody",
+                "_mappingEntityId": "annotationid"
```

Comment:
@vedg please take a note of these changes in the schema. We will need to add these in the newSchema as well.

Reply:
We should also discuss some changes to existing schema. Currently, we convert entityType name sent in vscode.dev url from portal to another entity type, which is used for making the dataverse call. And then we further map it to the enityType/Name we maintain in the schema. I believe we dont need to maintain different dataverse call names v/s schema names. Or, is there some background i am missing here. 
This simple change will simplify our pathParamToEntity conversion logic. Currently, we are using the same mapping object for both of the above conversions. Either we simplify the logic above, or atleast create two separate mapping objects.

Reply:
TODO: Leave a comment in code for this logic